slackbot/dispatcher.py

```diff
@@ -82,7 +82,7 @@ def filter_text(self, msg):
         else:
             m = AT_MESSAGE_MATCHER.match(text)
             if m:
-                msg['text'] = m.groups(2)
+                msg['text'] = m.groups(2)[1]
```

Comment:
It should be `m.group(2)`. `m.groups()` returns a tuple of all the captured parts, while `m.group(n)` returns the n-th capture.